src/components/tooltip/tooltip.spec.js

```diff
@@ -275,83 +274,68 @@ describe('<md-tooltip> directive', function() {
       expect($rootScope.testModel.isVisible).toBe(false);
     });
 
-    it('should not be visible on mousedown and then mouseleave', inject(function($document) {
-      buildTooltip(
-        '<md-button>' +
-         'Hello' +
-         '<md-tooltip md-visible="testModel.isVisible">' +
-            'Tooltip' +
-          '</md-tooltip>' +
-        '</md-button>'
-      );
-
-      // Append element to DOM so it can be set as activeElement.
-      $document[0].body.appendChild(element[0]);
-      element[0].focus();
-      triggerEvent('focus,mousedown');
-
-      expect($document[0].activeElement).toBe(element[0]);
-      expect($rootScope.testModel.isVisible).toBe(true);
-
-      triggerEvent('mouseleave');
-      expect($rootScope.testModel.isVisible).toBe(false);
-
-      // Clean up document.body.
-      $document[0].body.removeChild(element[0]);
-    }));
-
-    it('should not be visible when the window is refocused', inject(function($window, $document) {
-      buildTooltip(
-        '<md-button>' +
-         'Hello' +
-         '<md-tooltip md-visible="testModel.isVisible">' +
-            'Tooltip' +
-          '</md-tooltip>' +
-        '</md-button>'
-      );
-
-      // Append element to DOM so it can be set as activeElement.
-      $document[0].body.appendChild(element[0]);
-      element[0].focus();
-      triggerEvent('focus,mousedown');
-      expect(document.activeElement).toBe(element[0]);
-
-      triggerEvent('mouseleave');
-
-      // Simulate tabbing away.
-      angular.element($window).triggerHandler('blur');
-
-      // Simulate focus event that occurs when tabbing back to the window.
-      triggerEvent('focus');
-      expect($rootScope.testModel.isVisible).toBe(false);
-
-      // Clean up document.body.
-      $document[0].body.removeChild(element[0]);
-    }));
-
+    it('should not be visible on mousedown and then mouseleave',
+        inject(function($document) {
+          buildTooltip(
+            '<md-button>' +
+              'Hello' +
+              '<md-tooltip md-visible="testModel.isVisible">' +
+                'Tooltip' +
+              '</md-tooltip>' +
+            '</md-button>'
+          );
+
+          // Append element to DOM so it can be set as activeElement.
+          $document[0].body.appendChild(element[0]);
+          element[0].focus();
+          triggerEvent('focus,mousedown');
+
+          expect($document[0].activeElement).toBe(element[0]);
+          expect($rootScope.testModel.isVisible).toBe(true);
+
+          triggerEvent('mouseleave');
+          expect($rootScope.testModel.isVisible).toBe(false);
+
+          // Clean up document.body.
+          element.remove();
```

Comment:
Do you want to do something like this in the `afterEach` to be extra safe from someone forgetting to clean up?